src/Razor/Razor/src/IRazorUrlHelper.cs

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+using System.IO;
+using Microsoft.AspNetCore.Razor.TagHelpers;
+
+namespace Microsoft.AspNetCore.Razor
+{
+    public interface IRazorUrlHelper
+    {
+    }
+}
```

Comment:
I don't think we'll need to move this down to the Razor layer if we support inject.

Reply:
RazorInjectAttribute is currently in `Microsoft.AspNetCore.Mvc.Razor.Internal`